dashwellviz/figures.py

```diff
@@ -32,6 +32,15 @@ def __init__(self, n_tracks, shared_yaxes=True, **kwargs):
             **kwargs,
         )
 
+    def update_subplot_titles(self, titles_dict):
```

Comment:
I'd suggest perhaps renaming this to `update_track_titles`, and also the argument to `track_titles`, as it feels a little clearer?